tests/integration/accounts/test_contract_account.py

```diff
@@ -91,3 +93,27 @@ async def test_should_give_infinite_allowance_to_kakarot(
                 )
                 == "Uint256(low=340282366920938463463374607431768211455, high=340282366920938463463374607431768211455)"
             )
+
+        async def test_should_take_deployment_fees(
+            self, kakarot, eth, fund_evm_address
+        ):
+            evm_address = int(generate_random_evm_address(), 16)
+            computed_starknet_address = (
+                await kakarot.compute_starknet_address(evm_address).call()
+            ).result[0]
+
+            await fund_evm_address(evm_address)
+
+            assert (await eth.balanceOf(4).call()).result.balance.low == 0
```

Comment:
please assign magic number to a variable before usage, and use the variable instead

Reply:
resolved.